allspice/apiobject.py

```diff
@@ -795,40 +809,61 @@ def get_file_content(
         else:
             return [Content.parse_response(self.allspice_client, f) for f in self.allspice_client.requests_get(url, data)]
 
-    def get_generated_json(self, content: Union["Content", str], ref: Optional["Ref"] = None) -> dict:
+    def get_generated_json(
+            self,
+            content: Union[Content, str],
+            ref: Optional[Ref] = None
+    ) -> dict:
         """
         Get the json blob for a cad file if it exists, otherwise enqueue
         a new job and return a 503 status.
 
-        Note: This is still experimental and not yet recommended for
-        critical applications.
+        !!!WARNING: Here be dragons!!!
+        This is still experimental and not recommended for critical applications.
+        The structure and content of the returned dictionary can change at any
+        time. If you are using this, be prepared for things to silently break
+        at any time!
 
         See https://hub.allspice.io/api/swagger#/repository/repoGetAllSpiceJSON
         """
 
         if isinstance(content, Content):
             content = content.path
 
-        url = f"/repos/{self.owner.username}/{self.name}/allspice_generated/json/{content}"
+        url = self.REPO_GET_ALLSPICE_JSON.format(
+            owner=self.owner.username,
+            repo=self.name,
+            content=content,
+        )
         data = Util.data_params_for_ref(ref)
         return self.allspice_client.requests_get(url, data)
 
-    def get_generated_svg(self, content: Union["Content", str], ref: Optional[str] = None) -> bytes:
+    def get_generated_svg(
+            self,
+            content: Union[Content, str],
+            ref: Optional[Ref] = None
+    ) -> bytes:
         """
         Get the svg blob for a cad file if it exists, otherwise enqueue
         a new job and return a 503 status.
 
         Note: This is still experimental and not yet recommended for
-        critical applications.
+        critical applications. The content of the returned svg can change
+        at any time. If you are using this, be prepared for things to
```

Comment:
Again, I think we can remove this last line. The content of the SVG will change but its unlikely that the structure will change given how much of our frontend code relies on its structure. I think we are probably going to move to webgpu before the general SVG structure changes enough to matter.

Reply:
Softened both of these warnings.